brands/amenity/bank.json

```diff
@@ -28,7 +33,8 @@
       "brand": "ABSA",
       "brand:wikidata": "Q331688",
       "brand:wikipedia": "en:ABSA Group Limited",
-      "name": "ABSA"
+      "name": "ABSA",
+      "official_name": "Absa Group Limited"
```

Comment:
I would not add this one - it is almost certainly owner of the bank rather than official bank name, and something not worth tagging.

Reply:
Probably a good call.